PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/FontColorFormat.cs

```diff
@@ -1,17 +1,18 @@
 ﻿using System;
 using System.Drawing;
-
-using Microsoft.Office.Interop.PowerPoint;
+using Microsoft.Office.Core;
 using PowerPointLabs.ActionFramework.Common.Log;
 using PowerPointLabs.Utils;
+using Shape = Microsoft.Office.Interop.PowerPoint.Shape;
+using Shapes = Microsoft.Office.Interop.PowerPoint.Shapes;
 
 namespace PowerPointLabs.SyncLab.ObjectFormats
 {
     class FontColorFormat
     {
         public static bool CanCopy(Shape formatShape)
```

Comment:
Doesn't seem like this function is used anywhere..?

Reply:
Format object methods are accessed with reflection so the ide doesn't pick it up. See Format.cs's CanCopy method. Not sure if we will be refactoring this some time down the line to an oop approach. Either that or some comments in a README